content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -345,6 +367,8 @@ Cleanup the `cluster2` cluster:
 $ kubectl delete --context=$CTX_CLUSTER2 -f istio-remote-auth.yaml
 $ kubectl delete --context=$CTX_CLUSTER2 ns istio-system
 $ kubectl delete --context=$CTX_CLUSTER2 ns sample
+$ unset CLUSTER_NAME SERVER SECRET_NAME CA_DATA TOKEN INGRESS_HOST SECURE_INGRESS_PORT INGRESS_PORT
```

Comment:
+ CTX_CLUSTER2

Reply:
Fixed in https://github.com/istio/istio.io/pull/3428/commits/880a49c8d0b86d86ef0d0194acc6d2ffe4064486